src/rollback_to_stable/rts_btree.c

```diff
@@ -167,9 +183,11 @@ __rts_btree_abort_insert_list(WT_SESSION_IMPL *session, WT_PAGE *page, WT_INSERT
                  */
                 WT_ERR(__wt_rts_history_delete_hs(session, key, rollback_timestamp + 1));
         }
+    __wt_scr_free(session, &key_string);
```

Comment:
As we are freeing the same variable at line 190 and we are not going to return in between, there is no need for this free statement.

Reply:
Ah yes didn't realise, thanks! [534d8af](https://github.com/wiredtiger/wiredtiger/pull/8960/commits/534d8af32c40fedbd4e238cd1d0c7fc73aed1bb9)